<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="InputString">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>131</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
     </font>
    </property>
   </widget>
   <widget class="QTableWidget" name="AnalyseProcess">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>611</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="InputRules">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
     </font>
    </property>
   </widget>
   <zorder>InputRules</zorder>
   <zorder>InputString</zorder>
   <zorder>AnalyseProcess</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionAnalyse"/>
   </widget>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAnalyse">
   <property name="text">
    <string>Analyse</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
